src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/AbstractFormStore.js

```diff
@@ -190,6 +200,10 @@ export default class AbstractFormStore
         this.updateFieldPathEvaluations();
     }
 
+    setMultiple() {
```

Comment:
this feels a bit strange.. why not just call `updateFieldPathEvaluations()` from the child class?

Reply:
I prefer this over calling `updateFieldPathEvaluations` in the child classes... There is already the `MemoryFormStore` and the `ResourceFormStore`, so we would have to do that at two different places. Which gets even worse, if we realize that there is more code we want to run when multiple properties are changed.

Reply:
alright, dont have a strong opinion 🙂